ray-operator/controllers/ray/utils/validation_test.go

```diff
@@ -201,30 +229,44 @@ func TestValidateRayClusterSpecGcsFaultToleranceOptions(t *testing.T) {
 			expectError:              true,
 			errorMessage:             errorMessageExternalStorageNamespaceConflict,
 		},
+		{
```

Comment:
These two tests have already been tested in `TestValidateRayClusterSpecRedisUsername`.